filters/include/pcl/filters/crop_hull.h

```diff
@@ -130,17 +130,12 @@ namespace pcl
       }
 
     protected:
-      /** \brief Filter the input points using the 2D or 3D polygon hull.
-        * \param[out] output The set of points that passed the filter
-        */
-      void
-      applyFilter (PointCloud &output) override;
 
       /** \brief Filter the input points using the 2D or 3D polygon hull.
         * \param[out] indices the indices of the set of points that passed the filter.
         */
       void        
-      applyFilter (Indices &indices) override;
+      applyFilter (Indices &indices) final;
```

Comment:
This is a breaking change, as someone external could inherit this class and override it.

Reply:
Ok, I will revert it.